src/backend/src/prisma/schema.prisma

```diff
@@ -392,6 +425,8 @@ model Project {
   car          Car            @relation(fields: [carId], references: [carId])
   teams        Team[]         @relation(name: "assignedBy")
   favoritedBy  User[]         @relation(name: "favoritedBy")
+  Graph        Graph?         @relation(fields: [graphLinkId], references: [linkId])
```

Comment:
Maybe I'm a bit confused about the purpose of this field, but shouldn't it be a many to many relation?